dev/InfoBar/InfoBar_themeresources.xaml

```diff
@@ -0,0 +1,103 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="InfoBarCriticalSeverityStatusBrush" Color="#FDE7E9"/>
+            <SolidColorBrush x:Key="InfoBarWarningSeverityStatusBrush" Color="#FFF4CE"/>
+            <SolidColorBrush x:Key="InfoBarSuccessSeverityStatusBrush" Color="#DFF6DD"/>
+            <SolidColorBrush x:Key="InfoBarInformationalSeverityStatusBrush" Color="#F2F2F2"/>
+
+            <StaticResource x:Key="InfoBarHyperlinkButtonForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
+
+            <StaticResource x:Key="InfoBarBorderBrush" ResourceKey="SystemControlTransparentBrush" />
+            <Thickness x:Key="InfoBarBorderThickness">0</Thickness>
+        </ResourceDictionary>
+        
+        <ResourceDictionary x:Key="Dark">
+            <SolidColorBrush x:Key="InfoBarCriticalSeverityStatusBrush" Color="#442726"/>
+            <SolidColorBrush x:Key="InfoBarWarningSeverityStatusBrush" Color="#433519"/>
+            <SolidColorBrush x:Key="InfoBarSuccessSeverityStatusBrush" Color="#393D1B"/>
+            <SolidColorBrush x:Key="InfoBarInformationalSeverityStatusBrush" Color="#2B2B2B"/>
```

Comment:
@gabbybilka There's been some sadness over the colors, especially the dark ones.

Reply:
I'll check in with Kim, what feedback is there for the light themed ones?

Reply:
They seem fine!

Reply:
We're going to stick with the muted background colors here to keep the focus on the text & icon due to the variability of color interpretation (i.e. color blindness & meaning across cultures). Thanks for pointing this out though!

Reply:
Are all background resources to stay the same? I found the success green very hard to identify in dark theme, maybe I am not the only one on that.

Reply:
That success color is the same color used by [Fluent UI's MessageBar](https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar):

![image](https://user-images.githubusercontent.com/1398851/94957483-1f5f5100-04ee-11eb-8cd4-ba1cdc82792d.png)
Paired with the colored icon, I think that looks ok. That said, I do like the used green background color for the Light theme a bit more (I kinda feels it's a more "natural" green color to convey success than the green background color used in dark theme):

![image](https://user-images.githubusercontent.com/1398851/94957647-63eaec80-04ee-11eb-9985-7e6ccb40d90e.png)

Since Fluent UI ships with these colors, I expect considerable design thoughts already went into these colors here...and the colored icon does help quite a bit. However, if I were to take away the colored icon (and have it use black/white foreground color), I would feel the used background colors to be a bit lacking....

If we keep the current background colors, how about changing the icon foregrounds to match the ones used by Fluent UI's MessageBar?

Reply:
The icon color definitely helps a lot here to convey success. With the icons, it's a lot clearer what the InfoBar is conveying in my opinion.

Reply:
We explored using the icon colors from MessageBar but felt the brightly colored icons strayed from the Windows overall design language and principles. There's also an inconsistency in MessageBar with how the 'Warning' style in dark mode is the same color as 'Info', we didn't want to replicate that here. 

I think that the recommendation by @Felix-Dev to include the icon theme resources was a great one, and that if applications wanted to brighten the icons they could via those.

Reply:
@gabbybilka @teaP if we need more changes to colors/icons, lets create an issue and follow up with a fix.